code/unclassified/src/josephus_problem/README.md

```diff
@@ -0,0 +1,22 @@
+# Josephus Problem  
```

Comment:
You aren't following our documentation guide; All of the markdown should be formatted exactly as outlined in [this article](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md).

The markdown should be (Where you replace important info):

```Markdown
# Algorithm Name
Description

## Explanation
Pictorial diagram to explain the algorithm 

> Image credits: (name of the site/organisation from where the image was referenced)

## Algorithm
Algorithm as a numbered list.


## Complexity
Time and space complexity

---
<p align="center">
	A massive collaborative effort by <a href="https://github.com/OpenGenus/cosmos">OpenGenus Foundation</a> 
</p>

---
```